tests/e2e/rtu/test_execution.py

```diff
@@ -11,18 +11,18 @@
 
 
 async def test_single_cell(api_client: APIClient, notebook_maker):
-    notebook = Notebook(cells=[CodeCell(id='cell_1', source='print("hello world"); 2 + 2')])
+    notebook = Notebook(cells=[CodeCell(id="cell_1", source='print("hello world"); 2 + 2')])
     file: File = await notebook_maker(notebook=notebook)
     # TODO: remove sleep when Gate stops permission denied on newly created files (db time-travel)
     await asyncio.sleep(2)
 
     rtu_client: RTUClient = await api_client.connect_realtime(file)
     assert rtu_client.builder.nb.cells == notebook.cells
 
-    kernel_session: KernelSession = await api_client.launch_kernel(file.id)
+    kernel_session: KernelSession = await api_client.launch_kernel(file.id, kernel_name="python3")
     await rtu_client.wait_for_kernel_idle()
 
-    queued_execution = await rtu_client.queue_execution('cell_1')
+    queued_execution = await rtu_client.queue_execution("cell_1")
```

Comment:
new Black respelled the quote style here, sorry.

Reply:
yeah all of tests/ needs to have black applied post-double-quote-policy-change